docs/guides/snowflake_integration.md

```diff
@@ -486,3 +505,128 @@ FN_T_UID2_IDENTITY_MAP(EMAIL_HASH, 'email_hash')
+- For data providers: `DP.FN_UID2_ENCRYPT`
+
+>NOTE: The raw UID2 value is a Base64-encoded string, 44 characters in length.
+
+#### FN_UID2_ENCRYPT: Syntax
+
+The following is the basic syntax for the encrypt function:
+
+```
+Function: FN_UID2_ENCRYPT(raw_uid VARCHAR(44))
+RETURNS VARCHAR(220)
+```
+
+#### FN_UID2_ENCRYPT: Example
+
+The following example illustrates use of this function. The actual change varies according to the specific database, so the `WHERE` is not specified in the example.
+
+```
+SELECT T.token, LATERAL (FN_UID2_DECRYPT(T.token)) AS raw_uid
+FROM receiver's_table T
+WHERE ...
+```
+
+Result: An additional column named `raw_uid` is added to the query result.
+
+```
++------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| raw_uid                                                          | token                                                                                                                                                                                                                        |
+|------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF | AAMAAAACAAECAwQFBgcICQoL18RtKOdx1LlG2zgxK7bTW/nomJIeMFyEJOAMaer+20FTzLBNCJRajrXzXWzb9H3FCtTXs5Rl8PJhYnxxWmMtQ11VHfjI7XiA71CctX2vvCBzD6VEszdNq/t66B1cvLKVjoq2lijm6yyaRi0ovJPuk2dcb8dm4dkHMWGkkOrXbPPvrpsIvYQHH11sbqVycYJZm5w= |
++------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+### FN_UID2_DECRYPT
+
+This function decrypts a UID2 token and returns the corresponding raw UID2.
+
+Use the applicable prefix to indicate your role:
+- For advertisers: `ADV.FN_UID2_DECRYPT`
+- For data providers: `DP.FN_UID2_DECRYPT`
+
+#### FN_UID2_DECRYPT: Syntax
+
+The following is the basic syntax for the decrypt function:
+
+```
+Function: FN_UID2_DECRYPT(token VARCHAR(220))
+RETURNS /*raw uid*/ VARCHAR(44)
+```
+
+####  FN_UID2_DECRYPT: Example
+
+The following example illustrates use of this function. The actual change varies according to the specific database, so the `WHERE` is not specified in the example.
+
+```
+SELECT T.token, LATERAL (FN_UID2_DECRYPT(T.token)) AS raw_uid
+FROM receiver's_table T
+WHERE ...
+```
+
+Result: An additional column named `token` is added to the query result.
```

Comment:
`token` should be `raw_uid`.

Reply:
@mcollins-ttd : Updated this and also the encrypt example line 548 please check both.